Documentation/push-templates/templates/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Adobe Experience Platform SDK push templates
+description: Learn about the push templates provided and supported by the Adobe Campaign Classic Mobile SDK extension.
+keywords:
+- Adobe Campaign Classic
+- Push
+- Push Template
+- Push Templates
+---
+
+# Adobe Experience Platform - push templates
+
+This document outlines the push templates that are supported by default.
+
+<InlineAlert variant="info" slots="text"/>
+
+| Template Name | Compatibility | Description | Sample |
```

Comment:
In the compatibility column here, do you want to mentioned NotificationBuilder 3.0.0 instead?